specification/cdn/resource-manager/Microsoft.Cdn/2016-10-02/cdn.json

```diff
@@ -343,6 +343,47 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Cdn/profiles/{profileName}/getSupportedOptimizationTypes": {
```

Comment:
@ravbhatnagar That is true. We should not have any issues with access control because all the CDN users can make get/put/patch calls.
I believe, we're now waiting for the ARM team to sign-off.